pkg/rtc/mediatrack.go

```diff
@@ -61,9 +61,14 @@ func NewMediaTrack(params MediaTrackParams) *MediaTrack {
 	t := &MediaTrack{
 		params: params,
 		dynacastManager: NewDynacastManager(DynacastManagerParams{
```

Comment:
question, would it be simpler to just not initialize dynacastManager if it's an audio? and similarly for MediaLossProxy?

Reply:
Yeah, can do that. Needed a few ifs in the top level module which I thought was a bit more hard to read, but I prefer that too. Let me try that again.

Reply:
Done in this commit @davidzhao

Reply:
that was.. so freaking fast.. insane 🤯